fastly/resource.go

```diff
@@ -81,12 +81,12 @@ func (c *Client) ListResources(i *ListResourcesInput) ([]*Resource, error) {
 
 // CreateResourceInput is used as input to the CreateResource function.
 type CreateResourceInput struct {
-	// Name is the name of the resource being linked to (e.g. an object store).
+	// Name is the name of the resource being linked to (e.g. an kv store).
```

Comment:
"an" is used before a word starting with a vowel, so with this change we should use "a" instead...

```suggestion
	// Name is the name of the resource being linked to (e.g. a kv store).
```